.Documentation/articles/getting-started/designer.md

```diff
@@ -81,7 +81,7 @@ A transition may contain multiple conditions. With conditions, you can define wh
 
 #### Training Course Creation Wizard
 
-To create a training course, select the `Innoactive > Creator > Create New Course...` menu option. This will open the Training Creation Wizard. Type the name of your training and press the `Create` button.
+To create a training course, in the top menu bar, select the `Innoactive > Creator > Create New Course...` menu option. This will open the Training Creation Wizard. Type the name of your training and press the `Create` button.
```

Comment:
```suggestion
To create a training course, in the top menu bar, select the `Innoactive > Create New Course...` menu option. This will open the Training Creation Wizard. Type the name of your training and press the `Create` button.
```

Reply:
Can you check in the whole file where the path to the menu is still wrong? Thank you

Reply:
good catch! will do 👍

Reply:
I have checked this for every menu option, I am think I found them all